articles/active-directory/develop/scenario-protected-web-api-app-configuration.md

```diff
@@ -137,6 +137,10 @@ The JwtBearer middleware, like the OpenID Connect middleware in web apps, is dir
 
 There can also be special validations. For example, it's possible to validate that signing keys (when embedded in a token) are trusted and that the token isn't being replayed. Finally, some protocols require specific validations.
 
+## Token validation in Azure Functions
```

Comment:
@maliksahil, I'd put this paragraph after Validators (otherwise we have an H2 here, and an unrelated H3 after?